src/components/AddBook.js

```diff
@@ -1,4 +1,8 @@
+/* eslint-disable import/extensions */
 import { v4 as uuidv4 } from 'uuid';
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook } from '../redux/books/books.js';
```

Comment:
Hello @BrianSammit, I was having problems with this too, in my local machine is running fine, I always wanted to say that, but definitely, something is wrong since you are getting a different path. I'll delete all my local files, and clone the repo again, figure what's wrong and then rename properly this folder to lowercase, the Uppercase in there was a mistake that I made from the beginning. Thank you for pointing it out 😅

Reply:
![image](https://user-images.githubusercontent.com/85680436/135633552-5c759b50-34a0-4416-b238-3ccf85fcda7e.png)